### ISSUE TITLE
Playback stops after 60 seconds

### ISSUE THREAD

**Issue by itniels** [miniwa/debil-bot#171]
Seems the bot only plays for 60 seconds, I have seen other bots do this as well in my testing today, so probably not specific to this, not sure if they have made an update to how bots can transmit audio or something along those lines.

**Comment by Ozak93** [comment]
Yep facing this issue aswell, i have to manually /leave and /join and it'd join playing audio
